src/test/java/seedu/guest/model/guest/EmailTest.java

```diff
@@ -65,4 +67,11 @@ public void isValidEmail() {
         assertTrue(Email.isValidEmail("if.you.dream.it_you.can.do.it@example.com")); // long local part
         assertTrue(Email.isValidEmail("e1234567@u.nus.edu")); // more than one period in domain
     }
+
+    @Test
+    public void hashcode() {
```

Comment:
Similarly, should we standardise the format for these tests with line spacing and comments?

Reply:
Agreed. Added line spacing and comments.